src/decoder/lattice-faster-decoder.cc

```diff
@@ -3,6 +3,7 @@
 // Copyright 2009-2012  Microsoft Corporation  Mirko Hannemann
 //           2013-2014  Johns Hopkins University (Author: Daniel Povey)
 //                2014  Guoguo Chen
+//                2018  SJTU Zhehuai Chen
```

Comment:
You probably should have just your name-- affiliation is only normally required for employees.

One other point: please be careful with the 80-character limit.  We try to respect it except in exceptional circumstances.